lib/tasks/vital_model_checker.rake

```diff
@@ -7,11 +7,7 @@ task vital_model_checker: :environment do
 
   Sentry.capture_message('No Downloads recorded within the last 12 hours. Please check.') if download_count == 0
 
-  ratings_count = Rating.where(updated_at: (Time.now - 24.hours)..Time.now).count
+  ratings_count = Rating.where(updated_at: (Time.now - 7.days)..Time.now).count
 
   Sentry.capture_message('No Ratings recorded within the last 24 hours. Please check.') if ratings_count == 0
```

Comment:
Could we just update the message to reflect the change to 7 days too, thanks

Reply:
resolved thank you